services/app-api/src/resolvers/healthPlanPackage/createHealthPlanPackage.test.ts

```diff
@@ -3,6 +3,7 @@ import CREATE_HEALTH_PLAN_PACKAGE from '../../../../app-graphql/src/mutations/cr
 import { constructTestPostgresServer } from '../../testHelpers/gqlHelpers'
 import { latestFormData } from '../../testHelpers/healthPlanPackageHelpers'
 import { testCMSUser } from '../../testHelpers/userHelpers'
+import { testLDService } from '../../testHelpers/launchDarklyHelpers'
 
 describe('createHealthPlanPackage', () => {
     it('returns package with unlocked form data', async () => {
```

Comment:
Is it possible for us to reuse these tests? Run them twice, once with the feature flag and once without?

Reply:
I think we need to figure out how to, because thats what we have said is our definition of done.

Reply:
I'll let someone else take over figuring out to rerun these tests since I'm out all next week. The rest should be good to go.

Reply:
@macrael @haworku Hello, I read the jest docs, and `describe.each` seems to do what we wanted. Probably could use some more work to make it reusable. Feel free to modify it, or if you have a better way you can replace it!